infra/tf/vpc/sg.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_security_group" "app_servers" {
+  name        = "app_servers"
+  description = "Allow inbound web traffic"
+  vpc_id      = "${module.vpc.vpc_id}"
+
+  ingress {
+    description = "Port 80 from world to application servers"
+    from_port   = 80
```

Comment:
@BrianGreenhill Can you add port 22 for SSH?

Reply:
Or @BrianGreenhill what's the best practice here?

Reply:
we can allow it from any ip as a starting point. a best practice that i've seen used in the past is to create a bastion host that can be used as a jump server to access business critical resources.

Reply:
@BrianGreenhill Okay sure, sounds good.